graphium/cli/train_finetune.py

```diff
@@ -105,6 +130,12 @@ def run_training_finetuning(cfg: DictConfig) -> None:
     with SafeRun(name="TRAINING", raise_error=cfg["constants"]["raise_train_error"], verbose=True):
         trainer.fit(model=predictor, datamodule=datamodule)
 
+    # Save validation metrics - Base utility in case someone doesn't use a logger.
+    results = trainer.callback_metrics
+    results = {k: v.item() if torch.is_tensor(v) else v for k, v in results.items()}
+    with fsspec.open(fs.join(output_dir, "val_results.yaml"), "w") as f:
```

Comment:
In case W&B is used, all these results are already logged to W&B, no? 

But yes, you could configure the `output_dir` through the config ([ref](https://hydra.cc/docs/configure_hydra/workdir/)) and have it be in the W&B dir.

Reply:
yeah, but you'll get the ability of downloading the yaml directly from wandb.